assignments/prototypes.js

```diff
@@ -14,15 +14,38 @@
   * name
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  
 */
 
+function GameObject(obj) {
+  this.createdAt = obj.createdAt;
+  this.name = obj.name;
+  this.dimensions = obj.dimensions;
+
+}
+
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game.`
+  
+}
+
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
 
+function CharacterStats(obj) {
+  GameObject.call(this, obj)
+  this.healthPoints = obj.healthPoints;
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype); 
+
```

Comment:
yes, now we have linked up the GameObject prototype with the CharacterStats prototype!